src/components/Navbar.tsx

```diff
@@ -8,17 +18,16 @@ export const Navbar = () => {
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+          <NavLink className={activeLinkClasses} to="/">
```

Comment:
You're passing the `activeLinkClasses` function directly as a className. However, `activeLinkClasses` requires an argument with `isActive` property to function correctly. You need to call `activeLinkClasses` with the appropriate argument. This also applies to line 26.